scripts/pvscheck.sh

```diff
@@ -1,12 +1,14 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
There were bashisms already, shellcheck suggested it via `In POSIX sh, 'local' is undefined. [SC2039]`.